Eshop.Persistence/ProductConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Eshop.Domain;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Eshop.Persistence
+{
+    internal class ProductConfiguration : IEntityTypeConfiguration<Product>
+    {
+        public void Configure(EntityTypeBuilder<Product> builder)
+        {
+            builder.HasNoKey();
```

Comment:
really we do not have a key? don't we have actually in category oposite?
we have new table and there is primary key Id - autoincremented - also in video I was briefly talking about it.
check CategoryConfiguration.cs